include/admin/PhorumInputForm.php

```diff
@@ -208,7 +208,7 @@ function show()
     {
         global $PHORUM;
 
-        if(count($this->_help)){
+        if (is_array($this->_help)) {
```

Comment:
not a valid change, you need to check count as well

```suggestion
        if (is_array($this->_help) && count($this->_help)) {
```